### ISSUE TITLE
Exception: UnicodeEncodeError

### ISSUE THREAD

**Issue by pwilczynskiclearcode** [jorgebastida/awslogs#44]
I fired `awslogs get my.log -s24h` and the following happened
```
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 552, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 505, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/pwilczynski/.virtualenvs/ansible/local/lib/python2.7/site-packages/awslogs/core.py", line 111, in consumer
    print(' '.join(output))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2202' in position 224: ordinal not in range(128)
```

**Comment by grim-luminal** [comment]
I was running into a similar problem with my logs. In my case the problem only came up when redirecting `stdout` to a non-TTY. Python 2.7 resets the encoding for stdout to the platform default (which is most likely ascii) when stdout is not connected to a TTY. 

I got around it by setting the `PYTHONIOENCODING` environment variable to `UTF-8`.

**Comment by bhoehl** [comment]
@jorgebastida 
this can be easily fixed in awslogs/core.py
you need to add 
import codecs

and add utf8 encoding to the stdout stream

class AWSLogs(object):

    ACTIVE = 1
    EXHAUSTED = 2
    WATCH_SLEEP = 2

    FILTER_LOG_EVENTS_STREAMS_LIMIT = 100
    MAX_EVENTS_PER_CALL = 10000
    ALL_WILDCARD = 'ALL'

    def __init__(self, **kwargs):
        UTF8Writer = codecs.getwriter('utf8')
        sys.stdout = UTF8Writer(sys.stdout)

**Comment by jorgebastida** [comment]
After reviewing which were the best way to solve these utf-8 issues I think the suggested solution at the end of (https://wiki.python.org/moin/PrintFails) makes the most sense.

These changes are now on master and I'm planning to release 0.4 to pypi in a bit.
https://github.com/jorgebastida/awslogs/commit/e36ca69ef6b0533ad1f022947fe3e1bf57e0719c

**Comment by jorgebastida** [comment]
Apparently, this is not such a good idea python3... https://travis-ci.org/jorgebastida/awslogs/jobs/108382946 :imp:

**Comment by jorgebastida** [comment]
This was fixed as part of 0.4
